Add standalone/jarmod instructions to the Installing Optifine page (#212)
PolyMC/polymc.github.io: src/wiki/getting-started/installing-optifine.md

Therefore, you should consider the use of OptiFine alternatives whenever possible. [See the wiki page about it](../install-of-alternatives).

## <img src="https://www.optifine.net/favicon.ico" height="20" /> Installing OptiFine standalone

> This method requires you either have the minecraft version you wish to install OptiFine on to be installed through the vanilla launcher or to recreate the folder structure somewhere on your system (like /.minecraft/versions/*1.18.2*/*1.18.2*.jar).

Reviewer: ```suggestion
This method requires you either have the Minecraft version you wish to install OptiFine on to be installed through the vanilla launcher or to recreate the folder structure somewhere on your system (like /.minecraft/versions/*1.18.2*/*1.18.2*.jar).
```

Go to <https://optifine.net/downloads> and find your respective Minecraft version (eg. 1.18.2).

Once found, click **mirror**. Now click the **download** button to download your OptiFine jar file.

Execute the OptiFine jar file and change the selected folder location if required, then hit the **"Extract"** button and save the mod file somewhere you'll remember.

> Now in the PolyMC launcher edit the instance you wish to install OptiFine on, open the **Version** tab and on the right click **Add to Minecraft.jar**, select the extracted OptiFine mod(the file ending in \_MOD.jar) and confirm.

Reviewer 2: ```suggestion
Now in PolyMC edit the instance you wish to install OptiFine on, open the **Version** tab and click **Add to Minecraft.jar**, select the extracted OptiFine jarmod (the file ending in \_MOD.jar) and confirm.
```
<img width="1141" alt="image" src="https://user-images.githubusercontent.com/83089242/188480349-c509cd3a-feb9-43a4-a179-2d9442a7ac94.png">

you don't have to right click

> OptiFine will now be copied to and installed as a jarmod for that instance so it is safe to delete the installer and extracted mod. 

Reviewer 2: ```suggestion
OptiFine will now be installed as a jarmod in that instance so you can remove the installer and extracted jarmod. 
```
